src/test/FuzzerImpl.cpp

```diff
@@ -1232,6 +1232,14 @@ std::array<OfferParameters, 17> constexpr orderBookParameters{{
     // No limit available after previous offer
     {4, AssetID(2), AssetID(), 1, 1, 1, true},
 
+    // small offers to test multiple offers to get taken in a single op
```

Comment:
I'm not sure this is really adding value. The above configurations already makes it pretty easy for multiple offers to be taken in a single operation, and this could be made even easier by reducing the size of the offer at the top of those books. Do you see something that I'm not seeing?

Reply:
Valid point. I initially took the approach of "more is better", but that doesn't really apply to fuzzing. Removed the offers.